cpp/benchmarks/synchronization/synchronization.cpp

```diff
@@ -21,26 +21,29 @@
 #include <rmm/cuda_stream_view.hpp>
 #include <rmm/device_buffer.hpp>
 
+void flush_device_L2_cache(rmm::cuda_stream_view stream)
```

Comment:
nvbench internally flushes l2 by default (https://github.com/NVIDIA/nvbench/blob/a171514056e5d6a7f52a035dd6c812fa301d4f4f/nvbench/detail/measure_cold.cuh#L120). No need to port this
```suggestion
```

Reply:
Awesome! thanks!